src/FakeClient/FakeClientMain.cpp

```diff
@@ -377,6 +381,7 @@ int main(int argc, char* argv[]) {
     ErrorMessageOr<std::vector<orbit_grpc_protos::ModuleInfo>> modules_or_error =
         orbit_module_utils::ReadModules(options.process_id);
     ORBIT_FAIL_IF(modules_or_error.has_error(), "%s", modules_or_error.error().message());
+    process.UpdateModuleInfos(modules_or_error.value());
```

Comment:
If `!options.enable_api && !calculate_frame_time`, we are passing a process without modules to `Capture`, which is fine but feels a tad wrong. Should we just move this part out of the `if` to the initialization of `process`? ...